android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/LocationModule.java

```diff
@@ -154,6 +174,7 @@ private void stopWatching() {
     }
 
     SmartLocation.with(mScopedContext).location().stop();
+    SmartLocation.with(mScopedContext).geocoding().stop();
```

Comment:
hmm, it basically removes a listener and I am not sure how to keep track if geocoding is running or not. But I surely have to move this above checking the location parameters